<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitiatedStrategy</class>
 <widget class="QDialog" name="InitiatedStrategy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="2">
     <widget class="QLabel" name="coinLabel">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Coin Label</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QDoubleSpinBox" name="fractionalBuyingPercentage">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="coinBalanceLabel">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>coinBalance</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="budgetLabel">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Budget</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="fractionalBuyingLabel">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Fractional Buying%</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="usdtCoinPriceLabel">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Coin Value (USDT): </string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QDoubleSpinBox" name="budgetInput">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="usdtBalanceLabel">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>USDT Balance</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="logsContainer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>821</width>
     <height>351</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="logsContainerContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>819</width>
      <height>349</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>821</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="logsContainerLayout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QLCDNumber" name="elapsedTime">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>270</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="elapsedTimeLabel">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>270</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Elapsed Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentCycleTimeLabel">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>330</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Current Cycle:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="currentCycleTime">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>330</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="deployStrategyButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>270</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Deploy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopStrategyButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>330</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeMenuButton">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>540</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="investedLabel">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>390</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Invested (USDT):</string>
   </property>
  </widget>
  <widget class="QLabel" name="profitLossLabel">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>450</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(42, 62, 58);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Profit/Loss (USDT):</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
